source/operator.rst

```diff
@@ -218,6 +218,60 @@ configuration options for the Percona XtraDB Cluster.
 +-----------------+-------------------------------------------------------------------------------------------+
 |                                                                                                             |
 +-----------------+-------------------------------------------------------------------------------------------+
```

Comment:
We are not adding other options to operator.rst ? Like spec.pxc.replicationChannels. Or they are in another PR?